src/webOAuthServer.ts

```diff
@@ -28,6 +28,8 @@ const messages = Messages.loadMessages('@salesforce/core', 'auth');
 // Server ignores requests for site icons
 const iconPaths = ['/favicon.ico', '/apple-touch-icon-precomposed.png'];
 
+const CODE_BUILDER_REDIRECT_URI = 'api/oauth2/salesforce/callback';
+
```

Comment:
@shetzel Sorry the callback URL changed a little to match the same format as the rest of our api.

in the connected app they are

```
https://codebuilder-platform-api.herokuapp.com/api/auth/salesforce/callback
https://codebuilder-platform-api-stage-c6fd5fccd5aa.herokuapp.com/api/auth/salesforce/callback
```